doc/src/contracts/service_registry.md

```diff
@@ -0,0 +1,112 @@
+# Service Registry
+
+The service registry keeps track of the pool of workers that are related to each service.
+The core functionalities, such as registering a new service, worker authorization and un-authorization can only be called 
+from a governance address. To become active and be eligible to participate in voting for a specific service, 
+each worker should first be authorized and the bond enough stake. Service registry is used by ampd, verifier, and multisig prover.
+
+
```

Comment:
Can we add a paragraph here talking about what exactly is a service? Versus a chain. Call out specifically that we plan to use one service, and all chains are within that service.

Reply:
added